### PR TITLE
seo improvements

### PR DESCRIPTION
**Pull request by programad**

### CODE REVIEW

File: packages/ignition/src/app/robots.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ const robots = (): MetadataRoute.Robots => {
     rules: {
       userAgent: "*",
       allow: "/",
-      disallow: "/api/"
+      disallow: ["/api/", "/_next/"]
     },
-    sitemap: `${serverEnv.NEXT_PUBLIC_APP_URL}/sitemap.xml`
+    sitemap: `${serverEnv.NEXT_PUBLIC_APP_URL}/sitemap_index.xml`
```
Comment (amorimjj) [inline packages/ignition/src/app/robots.ts:11]: Do we really need rename this? I'm afraid we're going to update few places and tools because of that.

**Review by amorimjj — COMMENTED** [review]
It's good in general, just a minor comment

File: packages/ignition/src/app/robots.ts (hunk 1 above)
Comment (programad) [reply packages/ignition/src/app/robots.ts:11]: Yes we need. It is a standard. The only place I remember right now is Google Search Console.

Comment (amorimjj) [reply packages/ignition/src/app/robots.ts:11]: ok!

**Review by amorimjj — APPROVED** [review]
code 👍

**Pull request merged**